react-ui-kit/src/common/Selects/interface/ISelectComponentItem.ts

```diff
@@ -0,0 +1,23 @@
+import { ReactNode } from 'react';
+
+import * as SelectConstants from '../constants';
```

Comment:
Можно SelectCurrency импортировать сразу, без SelectConstants:
```suggestion
import type { SelectCurrency  } from '../constants';
```